tests/parser/types/test_lists.py

```diff
@@ -547,6 +547,70 @@ def bar(_baz: Foo[3]) -> String[96]:
     assert c.bar(c_input) == "Hello world!!!!"
 
 
+def test_list_of_nested_struct_arrays(get_contract):
```

Comment:
are these functional tests required for this PR? or just addl array tests?

Reply:
these are additional array tests. should I also add tests for the generated ABI?